tests/SpecTests/DocumentsMatchConstraint.php

```diff
@@ -37,6 +37,8 @@
 use function get_debug_type;
 use function in_array;
```

Comment:
Looks like you need to run `phpcbf` on this file.

Reply:
Whoops, done. Thanks!